alibi/explainers/experimental/counterfactuals.py

```diff
@@ -1062,8 +1061,8 @@ def __init__(self,
             'method_opts': method_opts,
             'feature_range': feature_range,
             'framework': framework,
-        }
-        self._explainer = self._explainer_type(*explainer_args, **explainer_kwargs, **kwargs)
+        }  # type: Dict
+        self._explainer = self._explainer_type(*explainer_args, **explainer_kwargs, **kwargs)  # type: ignore
```

Comment:
Why do we ingnore the type here? Isn't this a `dict`?

Reply:
There was a reason for this...

Reply:
With no type info I get the following error: `alibi/explainers/experimental/counterfactuals.py:1065: error: Function gets multiple values for keyword argument "predictor"`

Reply:
It's because `predictor` is somehow present in both `explainer_args` and `kwargs`.

Reply:
The correct type (correct me if I'm wrong) would be `#type: _WachterCounterfactual`, but the problem persists, so I just originally ignored the type.

Reply:
This is a known bug so I would stick with `# type: ignore`: https://github.com/python/mypy/issues/6799

Reply:
Agreed.